Config.psm1

```diff
@@ -21,57 +21,74 @@ Import-Module "$localResourcesDir\ini.psm1"
 function Get-availableConfigOptionOptions {
     return @(
         @{"Name" = "wim_file_path"; "DefaultValue" = "D:\Sources\install.wim";
-          "Description" = "Wim file path."},
+          "Description" = "The location of the WIM file from the Windows ISO."},
         @{"Name" = "image_name"; "DefaultValue" = "Windows Server 2012 R2 SERVERSTANDARD";
-          "Description" = "Wim image name"},
+          "Description" = "This is the complete name of the Windows version that will be generated. 
+                           In order to find the possible options, use the Get-WimFileImagesInfo command 
+                           and look for the Name property."},
         @{"Name" = "image_path"; "DefaultValue" = "${ENV:TEMP}\win-image.vhdx";
-          "Description" = "Image path"},
+          "Description" = "The destination of the generated image."},
         @{"Name" = "virtual_disk_format"; "DefaultValue" = "VHDX";
-          "Description" = "Virtual disk format"},
+          "Description" = "Select between VHD, VHDX, QCOW2, VMDK or RAW formats."},
         @{"Name" = "image_type"; "DefaultValue" = "HYPER-V";
-          "Description" = ""},
+          "Description" = "This parameter allows to choose between MAAS, KVM and Hyper-V specific images."},
         @{"Name" = "disk_layout"; "DefaultValue" = "BIOS";
-          "Description" = ""},
+          "Description" = "This parameter can be set to either BIOS or UEFI."},
         @{"Name" = "product_key";
-          "Description" = ""},
+          "Description" = "The product key for the selected OS."},
         @{"Name" = "extra_features";
-          "Description" = ""},
+          "Description" = "An comma separated array of extra features that will be enabled on the resulting image.
+                           These features need to be present in the ISO file."},
         @{"Name" = "force"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "It will force the image generation when $RunSysprep is $False or the selected $SwitchName
+                           is not an external one. Use this parameter with caution because it can easily generate
+                           unstable images."},
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "If set to true, MaaSHooks will be installed."},
         @{"Name" = "zip_password";
-          "Description" = ""},
+          "Description" = "If this parameter is set, after the image is generated, 
+                           a password protected zip archive with the image will be created."},
         @{"Name" = "administrator_password"; "GroupName" = "vm"; "DefaultValue" = "Pa`$`$w0rd";
-          "Description" = ""},
+          "Description" = "This will be the administrator's password before the image starts the sysprep process."},
```

Comment:
This will be the Administrator user's, so that AutoLogin can be performed on the instance, in order to install the required products, updates and perform the generation tasks like sysprep.